README.md

```diff
@@ -19,4 +19,29 @@ input look like? How would this complexity change if you generated permutations
 randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
-most important part. Add your answer to this markdown file.
+most important part. Add your answer to this markdown file.  
+  
+I used the following source in order to help create my code:  
+https://www.geeksforgeeks.org/print-all-possible-permutations-of-an-array-vector-without-duplicates-using-backtracking/  
+
+ 
+**Note**: Please be aware that I was slightly confused about what was being asked  
+in the question. If I am missing anything (or something is wrong) please don't  
+hesitate to let me know and I will fix it as soon as possible. Thank you.  
+ 
+**ANSWER**:  
+In the case for this algorithm, the best case scenario would be if the the program  
+was given an already sorted graph. In this case, the complexity would only be  
+$Theta(n)$ due to the algorithm only having to go over them once to make sure that  
+they were properly sorted. Next up, the worst case scenario would be if the sorted  
+array was the last permutation generated. This would give the the sorting algorithm  
+a complexity of $Theta(n!)$, as that's how many tries it would take to find the  
+correct answer. $Theta(n!)$ would also be the average runtime for this algorithm.  
```

Comment:
There's $n!$ permutations, but you also need to check for each one whether the array is sorted. How would that affect the complexity?